pkgs/development/libraries/google-cloud-cpp/default.nix

```
@@ -106,16 +105,18 @@ stdenv.mkDerivation rec {
     lib.optionalString doInstallCheck (
       lib.optionalString (!staticOnly) ''
         export ${ldLibraryPathName}=${lib.concatStringsSep ":" additionalLibraryPaths}
-      '' + ''
-        export GTEST_FILTER="-${lib.concatStringsSep ":" excludedTests.cases}"
       ''
     );
 
   installCheckPhase = lib.optionalString doInstallCheck ''
     runHook preInstallCheck
 
-    # disable tests that contact the internet
-    ctest --exclude-regex '^(${lib.concatStringsSep "|" excludedTests.whole})'
+    # Disable any integration tests, which need to contact the internet.
+    # Also disable the `storage_benchmark_*` tests. With Protobuf < v23.x they
+    # require -DGOOGLE_CLOUD_CPP_ENABLE_CTYPE_WORKAROUND=ON. With
+    # Protobuf  >= v23.x they require They require setting
+    # -DGOOGLE_CLOUD_CPP_ENABLE_CTYPE_WORKAROUND=OFF
```

```suggestion
    # Disable any integration tests, which need to contact the internet.
    # Also disable the `storage_benchmark_*` tests. 
    # With Protobuf < 23.x they require -DGOOGLE_CLOUD_CPP_ENABLE_CTYPE_WORKAROUND=ON.
    # With Protobuf >= 23.x they require They require setting -DGOOGLE_CLOUD_CPP_ENABLE_CTYPE_WORKAROUND=OFF
```

Version numbers don't contain prefixes like v in the nixpkgs.
Also writing one sentence per line makes it easier to understand especially now that the before and after case are better comparable

---

Done. Please take another look.